AsyncDisplayKit/Details/Transactions/_ASAsyncTransaction.h

```diff
@@ -112,6 +133,27 @@ typedef NS_ENUM(NSUInteger, ASAsyncTransactionState) {
                              queue:(dispatch_queue_t)queue
                         completion:(asyncdisplaykit_async_transaction_operation_completion_block_t)completion;
 
+/**
+ @summary Adds an async operation to the transaction.  The execution block will be executed immediately.
+ 
+ @desc The block will be executed on the specified queue and is expected to complete asynchronously.  The block will be
+ supplied with a completion block that can be executed once its async operation is completed.  This is useful for
+ network downloads and other operations that have an async API.
+ 
+ WARNING: Consumers MUST call the completeOperationBlock passed into the work block, or objects will be leaked!
+ 
+ @param block The execution block that will be executed on a background queue.  This is where the expensive work goes.
+ @param priority Execution priority; Tasks with higher priority will be executed sooner
+ @param queue The dispatch queue on which to execute the block.
+ @param completion The completion block that will be executed with the output of the execution block when all of the
+ operations in the transaction are completed. Executed and released on callbackQueue.
+ */
+- (void)addAsyncOperationWithBlock:(asyncdisplaykit_async_transaction_async_operation_block_t)block
```

Comment:
@appleguy, again, the async variant was already there :) I didn't introduce it, just added method variants with priority.